app/src/test/java/protect/card_locker/LoyaltyCardViewActivityTest.java

```diff
@@ -2,6 +2,7 @@
 
 import android.app.Activity;
 import android.app.DatePickerDialog;
```

Comment:
I was expecting this to be a unused import now. But It still seems to be used elsewhere, which looks incorrect.

However, handling that is clearly out of scope for this PR. Just pointing it out. It's on my to-do list for a follow-up.

@theimpulson I had not gotten around to looking into the failing test, thanks for fixing!